src/services/encryptionService.js

```diff
@@ -1,38 +1,134 @@
 import api from "@api/api";
+import createHash from "@utils/user/create_hash";
 import initVodozemac, { Account } from "vodozemac-javascript";
-import { setEncryptedUser } from "@store/values/EncryptedUser";
+import localforage from "localforage";
+import showCustomAlert from "@utils/show_alert";
+import store from "@store/store";
+import { upsertUser } from "@store/values/Participants";
 
 class EncryptionService {
+  #encryptionSession = null;
+  #account = null;
+  #vodozemacInitialized = false;
+
   constructor() {
-    initVodozemac().then(() => {
-      console.log("[encryption] Vodozemac init Ok");
-    });
+    this.#initializeVodozemac();
+  }
+
+  async #initializeVodozemac() {
+    try {
+      await initVodozemac();
+      this.#vodozemacInitialized = true;
+      console.log("[encryption] Vodozemac initialized successfully");
+    } catch (error) {
+      console.error("[encryption] Failed to initialize Vodozemac", error);
+    }
+  }
+
+  hasEncryptedAccount() {
```

Comment:
could be just `hasAccount `.
This is a part of encrypted service, so we know that this is about encryption